src/platform/packages/shared/kbn-alerts-ui-shared/src/action_variables/action_variables.ts

```diff
@@ -214,6 +214,16 @@ export const AlertProvidedActionVariableDescriptions: Record<
       defaultMessage: 'The number of consecutive runs that meet the rule conditions.',
     }),
   },
+  rule: {
+    description: i18n.translate('alertsUIShared.actionVariables.ruleLabel', {
+      defaultMessage: 'The rule object.',
+    }),
+  },
+  status: {
+    description: i18n.translate('alertsUIShared.actionVariables.statusLabel', {
+      defaultMessage: 'The status of the alert.',
+    }),
+  },
```

Comment:
I think this type needed to be added because of @doakalexi 's demo. I expect this will soon be obsolete as we move separate the engines.